### PR TITLE
Application landing - Success confirmation message

### PR DESCRIPTION
**Pull request by BryceSlade**
[Loom Video](https://www.loom.com/share/0de76e7dd1304976ad33fb1f98362075)

## Description

Created two components, AppError and AppSuccess. Both components claim their own route and will display a confirmation message depending on a successful or failed post request. Mentee and Mentor put request - added routing syntax with useHistory to direct the user to either the AppError or App Success page. Added routes in src/index.js for these components.

## Contributors

@danielaescutia07
@tristanharrower 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [ ] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by danielaescutia07 — APPROVED** [review]
This will give users a clear understanding if submission was successful or not. Great job!

File: src/index.js [hunk 1]
```diff
@@ -71,12 +74,8 @@ function App() {
...
         <Route path="/apply/mentee" component={Mentee} />
         <Route path="/apply/mentor" component={Mentor} />
-        <Route
-          path="/pending"
-          component={() => (
-            <div>"Application Is Pending" Component goes here</div>
-          )}
-        />
+        <Route path="/apply/success" component={AppSuccess} />
+        <Route path="/apply/error" component={AppError} />
```
Comment (MrJacobSullivan) [inline src/index.js:78]: This looks really good!

File: src/components/pages/RoleSignup/Applications/Mentee.js [hunk 2]
```diff
@@ -47,14 +48,18 @@ const initialFormValues = {
...
+  let history = useHistory();
+
   const postNewAccount = async newAccount => {
     try {
       const response = await axios.post(
         `${process.env.REACT_APP_API_URI}/application/new/mentee`,
         newAccount
       );
+      history.push('/apply/success');
       console.log('post response', response);
```
Comment (MrJacobSullivan) [inline src/components/pages/RoleSignup/Applications/Mentee.js:60]: Could you delete this `console.log`? This must have been left in from a previous PR.

File: src/components/pages/RoleSignup/Applications/Mentor.js [hunk 3]
```diff
@@ -42,14 +43,18 @@ const initialFormValues = {
...
+  let history = useHistory();
+
   const postNewAccount = async newAccount => {
     try {
       const response = await axios.post(
         `${process.env.REACT_APP_API_URI}/application/new/mentor`,
         newAccount
       );
+      history.push('/apply/success');
       console.log(response);
```
Comment (MrJacobSullivan) [inline src/components/pages/RoleSignup/Applications/Mentor.js:55]: Could you delete this `console.log` as well?

File: src/components/pages/RoleSignup/Applications/Mentor.js [hunk 4]
```diff
@@ -42,14 +43,18 @@ const initialFormValues = {
 const Mentor = () => {
   const [formValues, setFormValues] = useState(initialFormValues);
 
+  let history = useHistory();
```
Comment (MrJacobSullivan) [inline src/components/pages/RoleSignup/Applications/Mentor.js:46]: Could this `let` be a `const`?

**Review by MrJacobSullivan — APPROVED** [review]
This looks really good! I especially like the route names you chose for both a successful application and an unsuccessful application. I added a couple suggestions for changes but they aren't critical to the functionality of this PR. Nice work!

**Review by tristanharrower — CHANGES_REQUESTED** [review]
The routing and components look good! Remove those console.logs that Jacob pointed out and it should be good to go!

File: src/components/pages/RoleSignup/Applications/Mentor.js (hunk 4 above)
Comment (BryceSlade) [reply src/components/pages/RoleSignup/Applications/Mentor.js:46]: Yes!

**Review by Fvaldenegro95 — APPROVED** [review]
Other than console.logs, This is clean!

**Review by BryceSlade — COMMENTED** [review]
Great styling!

**Review by tristanharrower — APPROVED** [review]
Routing and Styling look good, console.logs removed! GJ!

**Review by rdperry031 — APPROVED** [review]
Confirmation messages are essential for the user experience. Good job!

File: src/components/pages/RoleSignup/Applications/Mentee.js (hunk 2 above)
Comment (BryceSlade) [reply src/components/pages/RoleSignup/Applications/Mentee.js]: Now deleted.

**Comment by r-gio28** [comment]
great styling! the routing and components look good.

**Pull request merged**